### PR TITLE
adbund adapter

### PR DESCRIPTION
**Pull request by AdBund**
## Type of change

- [x] New bidder adapter

## Description of change

- test parameters for validating bids
```
{
  bidder: 'adbund',
  params: {
    sid: '110238',
    bidfloor: 0.036
  }
}
```
- contact email of the adapter’s maintainer: support@adbund.com
- [x] official adapter submission

### CODE REVIEW

**Comment by jaiminpanchal27** [comment]
@AdBund Please add unit tests

File: src/adapters/adbund.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  function _requestBids(bid) {
+    var info = {
+      referrer: utils.getTopWindowUrl(),
+      domain: utils.getTopWindowLocation().hostname,
+      ua: window.navigator.userAgent
+    };
+    var param = Object.assign({}, bid.params, info);
+    param.sizes = JSON.stringify(param.sizes || bid.sizes);
```
Comment (jaiminpanchal27) [inline src/adapters/adbund.js]: I am getting bid back for size [[300,250]] but not for [300,250]. Please add support for the same.

**Comment by adbundzheng** [comment]
This is a what's wrong? @jaiminpanchal27 

> Chromium 37.0.2062 (Ubuntu 0.0.0) ERROR
>   Uncaught ReferenceError: pbjs is not defined
>   at /home/travis/build/prebid/Prebid.js/test/spec/adapters/adbund_spec.js:8558

File: test/spec/adapters/adbund_spec.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import { expect } from 'chai';
+import Adapter from '../../../src/adapters/adbund';
+import adapterManager from 'src/adaptermanager';
```
Comment (jaiminpanchal27) [inline test/spec/adapters/adbund_spec.js]: import of adaptermanager is causing the problem. Remove this line. 
You can check https://github.com/prebid/Prebid.js/blob/master/test/spec/adapters/hiromedia_spec.js

File: src/adapters/adbund.js (hunk 1 above)
Comment (adbundzheng) [inline src/adapters/adbund.js]: Has been modified, please review again

**Comment by jaiminpanchal27** [comment]
Looks good. Thanks @AdBund

**Comment by jaiminpanchal27** [comment]
This is merged into master. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Comment by adbundzheng** [comment]
Added adbund. md to dev-docs directory.Thanks

**Pull request merged**